### PR TITLE
Added a force call to Require.js

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

File: Resources/views/base.html.twig [hunk 1]
```diff
@@ -26,7 +26,8 @@
                 "jquery": "{{ asset('bundles/fsiadmin/jquery-1.11.2.min') }}",
                 "bootstrap":  "{{ asset('bundles/fsiadmin/bootstrap.min') }}"
             }
-        });
+        });       
+        require(['jquery', 'bootstrap']);
```
Comment (chives) [inline Resources/views/base.html.twig]: I think it should be placed inside ``{% block javascripts %}`` at the bottom of the page body to delay real JS loading as much as possible

**Comment by szymach** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Comment by bartekgruszka** [comment]
:+1:

**Comment by szymach** [comment]
@fsi-open

**Comment by rn0** [comment]
:+1:

**Pull request merged**